backend/src/app.js

```diff
@@ -6,6 +6,7 @@ import fetch from 'node-fetch';
 import { legApi, membersFromYear } from './nysenate-api.js';
 import { categories, categoryMapping } from './categories.js';
 import { openStatesApi } from './openstates-api.js';
+import { mapBoxApi } from './mapbox-api.js';
```

Comment:
Added import of mapBoxApi from the new backend file. Should simply follow the same structure as other imports. Lmk if it doesn't!